### ISSUE TITLE
more documentation

### ISSUE THREAD

**Issue by anarcat** [wallabag/wallabagger#15]
it is unclear to me what exactly this app does: does it track what i am reading to update the reading point in articles? does it watch which pages are loaded to see if they are present in the database?

the github project header says "Chrome plugin for wallabag v2. The plugin for wallabag v1 is here https://github.com/wallabag/chrome-ext http://wallabag.github.io/wallabagger/"

this is a bit ambiguous to me: i originally discarded those two links because i thought they were both for v1. (does anyone still use v1 anyways?) maybe it would be better to put the link to the v1 version in the readme instead of the header, so that it's clear that the second link is useful documentation.

but then in that second link, the "usage" link is broken...

in general, i could find all this only after going on the chrome web store, and going on the home page. it would be nice to have a little more documentation in the plugin itself that explains what is going on, or at least a pointer to an online help...

**Comment by nicosomb** [comment]
I removed the old link in the github header. 

Is this documentation useful for you https://www.wallabag.org/blog/2016/11/18/wallabagger-howto ? 

We need to add it on our documentation website.

**Comment by anarcat** [comment]
that's great! one comment about it - if you are a new wallabag user, it's not obvious what the "checkmark" and "hearth" icons do. in the text, it says:

> set archived and starred flags icons These icons change its appeareance when the flags are set.

it doesn't say what the buttons actually do! the icons are also subtly different from the main interface. it seems that, in the main interface, when an article is "unread", there is a checkmark. when i click the checkmark, it turns into an arrow (which is a little weird to me: it looks like an undo icon of some sort, a bit confusing). in the chrome app, it's different: when the article is unread, the checkmark is "empty" and when it is read, it is full..

it would be nice to clarify what does what in there... although maybe there's a larger usability issue to discuss here...

thanks!

**Comment by SylChamber** [comment]
The page moved to [https://wallabag.org/en/news/wallabagger-howto](https://wallabag.org/en/news/wallabagger-howto).

**Comment by nicosomb** [comment]
We'll add it in our documentation.
